### ISSUE TITLE
`!pe` in windbg does not print first chance exception details

### ISSUE THREAD

**Issue by jkotas** [dotnet/runtime#100943]
## Repro

1. Start the following under windbg:
```
using System;

throw new Exception("My exception");
```
2. `sxe clr`
3. `g`
4. `!pe`

## Actual result

```
0:000> !pe
There is no current managed exception on this thread
```

## Expected result

`!pe` prints exception details.

**Comment by jkotas** [comment]
Works fine with `DOTNET_LegacyExceptionHandling=1`

**Comment by mangod9** [comment]
@janvorli, looks like this issue can be closed?

**Comment by janvorli** [comment]
Fixed by #100714
